pgx/src/datum/range.rs

```diff
@@ -67,6 +61,10 @@ impl<T> FromDatum for Range<T>
 where
     T: FromDatum + IntoDatum + RangeSubType,
 {
+    /// ## Safety
+    /// function requires that
+    /// - is_null is true
```

Comment:
Do you mean
```suggestion
    /// - is_null is correct
```
If this is a pointer based type, I don't know that we can ask for that invariant to be upheld because there are lots of places that these functions are called generically without knowledge of the underlying type's pointerness or valueness. I think we need to check the nullptrness ourselves.